packages/entity-list/src/components/Table/StyledTable.js

```diff
@@ -64,7 +64,7 @@ export const PaginationContainer = styled.div`
 
 const selectionStyles = css`
   display: contents;
-  cursor: pointer;
+  ${props => props.clickable && 'cursor: pointer;'}
```

Comment:
imo better looking with object destructuring: 
` ${({clickable}) =>clickable && 'cursor: pointer;'}`

Reply:
you're right dake 👍